### PR TITLE
Add build system support for using https://github.com/triSYCL/sycl

### PR DESCRIPTION
**Pull request by Ralender**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
 
-project(SYCL-path-tracer LANGUAGES CXX)
-
 # To run fast...
 set(CMAKE_BUILD_TYPE Release)
 # ... or to debug
 #set(CMAKE_BUILD_TYPE Debug)
 
 option(USE_SINGLE_TASK "Use a SYCL executor based on .single_task() instead of .parallel_for(), better for FPGA" OFF)
+set(SYCL_CXX_COMPILER "" CACHE STRING "Path to the SYCL compiler to use of not specified the triSYCL CPU implementation will be used" )
```
Comment (keryell) [inline CMakeLists.txt:9]: Some errors in the sentence?

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
+set(SYCL_DEVICE_TRIPLE "fpga64_sw_emu" CACHE STRING "Device triple to be used. only used with SYCL_CXX_COMPILER")
```
Comment (keryell) [inline CMakeLists.txt:10]:
```suggestion
# Use Xilinx FPGA software emulation by default
set(SYCL_DEVICE_TRIPLE "fpga64_sw_emu" CACHE STRING "Device triple to be used. Only used with SYCL_CXX_COMPILER")
```

**Review by keryell — CHANGES_REQUESTED** [review]
It does not work for me:
```<%>- cmake .. -DCMAKE_MODULE_PATH=~/Xilinx/Projects/Khronos/SYCL/triSYCL/cmake 
-- targeting: fpga64_sw_emu
-- The CXX compiler identification is unknown
CMake Error at CMakeLists.txt:23 (project):
  No CMAKE_CXX_COMPILER could be found.

  Tell CMake where to find the compiler by setting either the environment
  variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
  to the compiler, or to the compiler name if it is in the PATH.

-- Configuring incomplete, errors occurred!
```

**Comment by keryell** [comment]
Also you need to update the README to explain how to use the new configuration.

**Comment by Ralender** [comment]
> Also you need to update the README to explain how to use the new configuration.

at this time it is not yet possible to compile. with this recipe.

**Comment by keryell** [comment]
Thanks!

**Pull request merged**